docs/configuration/downstream_configuration_template.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Downstream configuration template
+sidebar_position: 2
+---
+
+This is a template for configuration of Packit's Fedora release automation (dist-git only).
+
+## How to use the template
+1. copy the content
+2. create a `packit.yaml`/`.packit.yaml` file in root directory of your dist-git repository in your `rawhide` branch
+and paste the content
+3. resolve the TODOs
+4. (optionally) run [`packit validate-config`](/docs/cli/validate-config) to make sure the configuration is valid
+
+## Template
+```yaml
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+# TODO fill in the URL to the upstream Git repo
+upstream_project_url: https://github.com/<upstream-repository>
```

Comment:
I'm not sure about the github.com example, it could confuse someone (even though GitHub is used by majority of upstreams).

How about:

```suggestion
# TODO fill in the URL to the upstream Git repo (one that can be passed to `git clone`, it should be read-only)
upstream_project_url: https://<upstream-git-repository-url>
```

or

```suggestion
# TODO fill in a URL you would use to anonymously clone the upstream Git repo (other VCS are currently not supported)
upstream_project_url: https://<upstream-git-repository-url>
```

Reply:
I rephrased it a little bit, but as for the example URL, I would stick with something more concrete

Reply:
PTAL